_docs/setup/kubernetes/quick-start.md

```diff
@@ -17,6 +17,7 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
 The following instructions assume you have access to a Kubernetes **1.7.4 or newer** cluster
 with [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
+If you wish to enable transparent injection of sidecar, you need to turn on Kubernetes alpha features in your cluster.
```

Comment:
Add a link to the automatic-sidecar-inject.html page

Reply:
there is already a " If your cluster has Kubernetes alpha features enabled,  ..." below - do we need to turn people away that don't have alpha features ?

Reply:
done, ptal